src/qt/sendcoinsdialog.cpp

```diff
@@ -324,6 +327,13 @@ void SendCoinsDialog::on_sendButton_clicked()
     questionString.append(QString("<span style='font-size:10pt;font-weight:normal;'><br />(=%2)</span>")
         .arg(alternativeUnits.join(" " + tr("or") + "<br />")));
 
+    if (ui->optInRBF->isChecked())
+    {
+        questionString.append("<hr /><span>");
+        questionString.append(tr("This transaction signals replaceability (optin-RBF)."));
```

Comment:
I still think this is likely to confuse users. Why must we say so here? (The alternative case seems much more of a liability...)